build-system/dep-check-config.js

```diff
@@ -46,6 +46,10 @@ exports.rules = [
     filesMatching: '**/*.js',
     mustNotDependOn: 'third_party/**/*.js',
     whitelist: [
+      // TODO(levitzky) remove next line when experiment
+      // 'a4a-adsense-use-new-format' is removed.
+      'tools/experiments/experiments.js->' +
+          'third_party/babel/custom-babel-helpers.js',
```

Comment:
Weird. I wanted to post the error message I got without this code, but once I removed it and re-ran dep-check, I no longer received any error. So I am removing this in my next push.